spec/models/post_spec.rb

```diff
@@ -26,23 +26,18 @@
     end
 
     it 'likes_counter is less than or eq to to 0' do
-      @post.likes_counter = -1
-      expect(@post).to_not be_valid
+      @post.likes_counter = 0
+      expect(@post).to be_valid
     end
 
     it 'comments_counter is greater than or eq to 0.' do
-      @post.comments_counter = -1
-      expect(@post).to_not be_valid
+      @post.comments_counter = 0
+      expect(@post).to be_valid
     end
 
```

Comment:
- [ ] Please add a scenario: when the coments_counter and likes_counter are less than 0 the @post should be invalid to ensure that the Post Model is tested well. 

Note: To avoid any confusion, I have verified the validations of your models and found that you mistyped "geater_than_or_equal_to", which should be "greater_than_or_equal_to". 